index.html

```diff
@@ -3,11 +3,119 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="/styles.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
+    <main>
+      <section class="content">
+        <article class="card">
+          <img
+            class="bookmark"
+            src="/assets/bookmark.svg"
+            alt="bookmark"
+            onclick=""
+          />
+          <!-- <svg class="bookmark" xmlns="http://www.w3.org/2000/svg">
+            <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16z" />
+          </svg> -->
+          <h2 class="card-headline">Comics</h2>
+          <div class="card-content">
+            <p class="card-text">When was Bruce Wayne born?</p>
+            <button class="answer-button">Show Answer</button>
+            <p class="answer-text" hidden>February 19th 1939</p>
+          </div>
+          <div>
+            <div class="tags">
+              <p class="item">#Comics</p>
+              <p class="item">#Fantasy</p>
+              <p class="item">#Batman</p>
+            </div>
+          </div>
+        </article>
+
+        <article class="card">
+          <img
+            class="bookmark"
+            src="/assets/bookmark.svg"
+            alt="bookmark"
+            onclick=""
+          />
+          <!-- <svg class="bookmark" xmlns="http://www.w3.org/2000/svg">
+            <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16z" />
+          </svg> -->
```

Comment:
To keep your code clean, it's best to decide for one variant: the `img` tag or the `svg`. The one you do not need, you should delete and not just comment out. :)